src/docs/product/sentry-basics/sampling/index.mdx

```diff
@@ -3,52 +3,25 @@ title: Sampling
 sidebar_order: 25
 redirect_from:
   - /product/sentry-basics/metrics/
-description: "Learn more about the different ways that you can sample the transaction events that you send to Sentry."
+description: "Learn more about Sentry samples your transaction data."
 ---
 
-<Include name="limited-avail-note.mdx" />
+Sampling is the process of taking a representative sample of your data to create an overall picture of the performance of your application. This isn’t typically necessary at lower volumes of data, but as volumes increase, it becomes less useful to store all data. That’s where sampling comes in.
 
-When you're using [sentry.io](https://sentry.io), the transaction information that you see is generated by one of two types of events:
+You can use your SDK sample rate to control how much and what type of data you send to Sentry. However, it can often be difficult to decide what an appropriate sample rate would be, and this can change over time. We recommend you send us all of your data and let us manage the sampling. Here’s why.
```

Comment:
```suggestion
You can use your SDK sample rate to control how much and what type of data you send to Sentry. However, it can often be difficult to decide what an appropriate sample rate would be, and this can change over time. We recommend you send us 100% of your data and let us manage the sampling. Here’s why.
```